utils/build-presets.ini

```diff
@@ -2463,6 +2463,9 @@ stdlib-deployment-targets=macosx-x86_64
 swift-primary-variant-sdk=OSX
 swift-primary-variant-arch=x86_64
 
+extra-cmake-options=
+   -DLLVM_ENABLE_Z3_SOLVER="NO"
```

Comment:
Shall we add this to the `tensorflow_linux` preset as well?

Reply:
Yeah, I think that we should keep them similar.  I really would prefer that you explicitly add this to the LLVM build, as the flags do not apply to all targets

Reply:
> I really would prefer that you explicitly add this to the LLVM build, as the flags do not apply to all targets

Does this mean making a change to `apple/llvm-project:swift/tensorflow`?

Reply:
No, it means that the flag should only be passed through to the llvm configuration rather than to all the cmake invocations.  This can be done in build-script-impl or in the preset.

Reply:
Done. I added `llvm-cmake-options=-DLLVM_ENABLE_Z3_SOLVER=NO` to `tensorflow_osx` and `tensorflow_linux` presets.